### PR TITLE
Add API conversion interface for Embedding layer.

### LINKED ISSUE #5623: Request for contributions: writing and testing API conversion decorators.
In Keras 2 (branch `keras-2`), the API for pretty much every layer has changed. We will keep supporting the Keras 1 API, and we will do so via **API conversion decorators**.

As an example, I have written the conversion decorator for the `Dense` layer. It was done in this commit: https://github.com/fchollet/keras/commit/5516c8fb42f01622c9a4a31c19cb46ae68cfa0f6
The decorator code is here: https://github.com/fchollet/keras/blob/keras-2/keras/legacy/interfaces.py#L15
The test for it is here: https://github.com/fchollet/keras/blob/keras-2/tests/keras/legacy/interface_test.py#L8

In order to launch Keras 2, we will need such decorators for every layer (and also for training routines and model constructors). If you want to contribute, please add these decorators (in `keras/legacy/interfaces.py`, with tests in `tests/keras/legacy/interface_test.py`). You just need to follow the template laid out with the `Dense` layer decorator.

You can leave out the `Conv*D` layers for now, since they will require special handling due to a positional/keyword argument conflict.

Thanks a lot!

### PR DESCRIPTION
**Pull request by rilut**
Fixes #5623.

### CODE REVIEW

**Comment by fchollet** [comment]
Please use the `convert_legacy_kwargs` utility in order to avoid code duplication: https://github.com/fchollet/keras/blob/keras-2/keras/legacy/interfaces.py#L15

File: keras/legacy/interfaces.py [hunk 1]
```diff
@@ -133,3 +133,50 @@ def wrapper(*args, **kwargs):
...
+        converted = []
+
+        # Remaining kwargs.
+        conversions = [
+            ('init', 'embeddings_initializer'),
+            ('W_regularizer', 'embeddings_regularizer'),
+            ('W_constraint', 'embeddings_constraint'),
+        ]
+
+        kwargs = convert_legacy_kwargs('Embedding', args, kwargs, conversions, converted)
```
Comment (fchollet) [inline keras/legacy/interfaces.py]: Should `args[1:]` here (the first entry in `args` is `self`).

Comment (rilut) [inline keras/legacy/interfaces.py]: @fchollet done. also done rebasing my commit

**Comment by fchollet** [comment]
Please fix the merge conflicts.

**Comment by rilut** [comment]
@fchollet ok, done rebasing my commit

**Comment by fchollet** [comment]
Please update this PR to use the new abstract interface (e.g. https://github.com/fchollet/keras/blob/keras-2/keras/legacy/interfaces.py#L76)

**Pull request closed without merging**